frontend/playwright.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
   forbidOnly: !!process.env.CI,
   retries: process.env.CI ? 2 : 0,
   workers: process.env.CI ? 1 : undefined,
-  reporter: 'html',
+  reporter: [['playwright-teamcity-reporter']],
```

Comment:
Would it make sense to keep `html`?
ex:
```
reporter: ['html', 'playwright-teamcity-reporter'],
```
(or whatever the correct syntax is)

Reply:
Maybe we  should use `process.env.CI` to use different reporters.

```ts
const reporter: ReporterDescription[] = [['list'], ['html']]

export default defineConfig({
  ...
  reporter: process.env.CI ? [...reporter, ['playwright-teamcity-reporter']]: reporter,
  ...
```